tools/uniswapv2/detect.py

```diff
@@ -200,9 +199,12 @@ def read_file_if_exists(name: str) -> Optional[Dict[str, Any]]:
             write_result_to_file(result, 'uniswap_lp_tokens_graph.json')
 
     if args.source in ('ethereum', 'both'):
+        msg_aggregator = MessagesAggregator()
+        database = DBHandler('fill', 'me', msg_aggregator, None)
```

Comment:
fill me? wtf? :octopus:

Reply:
again ... wtf it this? Why are you not answering all previous pings? I guess it was "fill" "me" before but if it's not needed then put it as "not", "needed"